### ISSUE TITLE
Amiga HDToolbox unable to access SD card (Pi4)

### ISSUE THREAD

**Issue by tstimac** [michalsc/Emu68#266]
Affected device: Pi4
Asset used: Emu68-pistorm32lite.zip from Jan 13

Followed these instructions step by step:
https://michalsc.github.io/Emu68/tutorials/SD_Preparation.html

Setting SCSI_DEVICE_NAME=brcm-emmc.device and starting HDToolbox shows an empty "Hard Drives in Sysyem" list.

Has the device name been changed recently?
Thanks!

**Comment by luvwagn** [comment]
if you re-open HDToolBox's icon info, is the SCSI_DEVICE_NAME still set correctly?  I can't remember which tool, but sometimes you need to edit the name and HIT ENTER, and then [Save], otherwise it doesn't "stick" ?
________________________________
From: tstimac ***@***.***>
Sent: Wednesday, February 28, 2024 5:19 PM
To: michalsc/Emu68 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [michalsc/Emu68] Amiga HDToolbox unable to access SD card (Pi4) (Issue #266)


Affected device: Pi4
Asset used: Emu68-pistorm32lite.zip from Jan 13

Followed these instructions step by step:
https://michalsc.github.io/Emu68/tutorials/SD_Preparation.html

Setting SCSI_DEVICE_NAME=brcm-emmc.device and starting HDToolbox shows an empty "Hard Drives in Sysyem" list.

Has the device name been changed recently?
Thanks!

**Comment by tstimac** [comment]
Yeah I know the press enter thing. Unfortunately it's not the issue. :(

**Comment by tstimac** [comment]
Addendum: With Kickstart 3.1 and Install 3.1 **it works**.

I downgraded my A1200 from 3.1 to 3.0 few years ago due to stability issues and faster boot time. For example one game (Hired Guns) did not work properly on 3.1. This was documented on English Amiga Board by Toni Wilen: https://eab.abime.net/showthread.php?p=841062#post841062

Still it would be interesting to see why it does not work on kickstart 3.0 / install 3.0 combo, nor it works on kickstart 3.0 / install 3.1 combo.

**Comment by michalsc** [comment]
Closing the issue since it may be related to install 3.0 tools and not emu68 itself
